ui/perfherder/Validation.jsx

```diff
@@ -74,10 +76,28 @@ const withValidation = ({ requiredParams }, verifyRevisions = true) => (
     };
 
     async checkRevisions(params) {
-      if (!params.originalRevision) {
+      let findOriginalRevision = params.originalRevision;
```

Comment:
nit: I'd change `findOriginalRevision` to `updatedOriginalRevision` or `upToDateOriginalRevision`